packages/gafl-webapp-service/src/pages/order-complete/order-complete/__tests__/order-complete.spec.js

```diff
@@ -2,89 +2,65 @@ import { LICENCE_DETAILS, NEW_TRANSACTION, ORDER_COMPLETE } from '../../../../ur
 import { addLanguageCodeToUri } from '../../../../processors/uri-helper.js'
 import { getData } from '../route.js'
 import { COMPLETION_STATUS, FEEDBACK_URI_DEFAULT } from '../../../../constants.js'
-import { displayStartTime } from '../../../../processors/date-and-time-display.js'
-import { LICENCE_TYPE } from '../../../../processors/mapping-constants.js'
-import { displayPermissionPrice } from '../../../../processors/price-display.js'
 
-jest.mock('../../../../processors/date-and-time-display.js')
 jest.mock('../../../../processors/uri-helper.js')
-jest.mock('../../../../constants.js', () => ({
-  ...jest.requireActual('../../../../constants.js'),
-  COMPLETION_STATUS: {
-    agreed: 'alright then',
-    posted: 'in the letterbox',
-    finalised: 'no going back now',
-    completed: 'all done'
-  },
```

Comment:
I've approved, but you've reduced the quality of the tests and reinforced the coupling between constants and the order complete getData function by removing this mocking. It's not a massive danger, as I doubt we're going to ever remove that, but that's why I did this.